js/interface.js

```diff
@@ -697,6 +559,129 @@ var DynamicLists = (function() {
         }
       });
     },
+    managePermissionList: function(isAddEntryActive, isEditEntryActive, isDeleteEntryActive) {
+      $('.add-entry-checkbox').find('.hidden-settings').toggleClass('active', isAddEntryActive);
+      $('.edit-entry-checkbox').find('.hidden-settings').toggleClass('active', isEditEntryActive);
+      $('.delete-entry-checkbox').find('.hidden-settings').toggleClass('active', isDeleteEntryActive);
+
+      this.toggleRuleType('insert', isAddEntryActive);
```

Comment:
Can we refactor this please?

```suggestion
      this.toggleRuleTypes({
        insert: isAddEntryActive,
        update: isEditEntryActive,
        delete: isDeleteEntryActive
      });
```